packages/backend-for-frontend/src/services/clientService.ts

```diff
@@ -387,17 +387,13 @@ export async function getSelfcareUserById(
   selfcareId: string,
   correlationId: CorrelationId
 ): Promise<selfcareV2ClientApi.UserResponse> {
-  try {
-    return selfcareClient.getUserInfoUsingGET({
-      params: { id: userId },
-      queries: { institutionId: selfcareId },
-      headers: {
-        "X-Correlation-Id": correlationId,
-      },
-    });
-  } catch (error) {
-    return {};
```

Comment:
your change seems legit, but which was the use case?

Reply:
IDK, but the way the code is written it would never have entered the catch